maquetado/index.html

```diff
@@ -15,33 +15,33 @@
     <h2 class="navbar-text">Developer Tools</h2>
   </div>
   <h2 class="title">Front-End Tools</h2>
-  <div class="technologies">
-    <div class="wrap">
+  <div class="container">
+    <div class="technologies">
       <div class="technology first-card">
         <img class="logo" src="./assets/react_icon.svg">
```

Comment:
Same as previos PRs. I would modify this into a more specific class like `tech-logo`